Backend/DataAccessLayer/Models/Reset.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccessLayer.Models
+{
+    public class Reset
+    {
+        [Required, Key] 
+        public string resetID { get; set; }
+
```

Comment:
ResetID will be a GUID

Reply:
I talked to Vong about using tokens for resetting the password. He said something along the lines of tokens not being necessary in this case because tokens are typically used when information stored in it needs to be passed around publicly. Since we have a table for resetting passwords, and all the information about the reset IDs is being stored there just further shows that the token isn't needed for resetting passwords.

I originally was using GUID to create the resetID, but Vong mentioned that it did have to be cryptographically secure so I'm using RNGCryptoServiceProvider to generate the resetIDs.

Reply:
I think you misunderstood - I'm not talking about a JWT. Simply about a token (a random string of characters).

There needs to be a separation of concerns here. We need two identifiers for each entry in the table. The first is the database ID that will be referenced by other tables. These are GUIDs across the entire application (see https://github.com/CECS-491A/SSO/blob/b4e0fd5c9500ddaae5afc93796b35c1390136787/Backend/DataAccessLayer/Models/User.cs#L13). The second is the token (again, random string) that will be sent to the user in the email (in link form, likely as a query parameter at the end of the link). This token _must_ be hard to guess. GUIDs are not very secure - they are fairly easily guessed.

The way you have it now, some other random person can guess the GUID and gain access to your account.